back-end/src/main/java/kr/co/ssalon/domain/service/MemberMeetingService.java

```diff
@@ -0,0 +1,28 @@
+package kr.co.ssalon.domain.service;
+
+import kr.co.ssalon.domain.entity.Meeting;
+import kr.co.ssalon.domain.entity.Member;
+import kr.co.ssalon.domain.entity.MemberMeeting;
+import kr.co.ssalon.domain.repository.MemberMeetingRepository;
+import org.apache.coyote.BadRequestException;
+
+import java.util.Optional;
+
+public class MemberMeetingService {
+
+    private MemberMeetingRepository memberMeetingRepository;
+
+    public MemberMeeting findByMemberAndMeeting(Member member, Meeting meeting) throws BadRequestException {
+
+        Optional<MemberMeeting> findMemberMeeting = memberMeetingRepository.findByMemberAndMeeting(member, meeting);
+        MemberMeeting memberMeeting = validationMemberMeeting(findMemberMeeting);
+        return memberMeeting;
+    }
+
+    private MemberMeeting validationMemberMeeting(Optional<MemberMeeting> findMemberMeeting) throws BadRequestException {
+        if (findMemberMeeting.isPresent()) {
+            return findMemberMeeting.get();
+        } else
+            throw new BadRequestException("멤버 미팅을 찾을 수 없습니다.");
```

Comment:
클라이언트에서는 '멤버 미팅'이 무엇을 뜻하는지 모를 수 있어서, 아래처럼 좀더 일반적인 문구로 써주면 좋을 것 같아요.
```suggestion
            throw new BadRequestException("회원이 참여하고 있는 모임을 찾을 수 없습니다.");
```

Reply:
아하 넵 변경하겠습니다